packages/sanity/src/core/releases/tool/BundlesOverview.tsx

```diff
@@ -16,7 +16,7 @@ type Mode = 'open' | 'archived'
 const EMPTY_BUNDLE_GROUPS = {open: [], archived: []}
 
 export default function BundlesOverview() {
-  const {data, loading} = useBundlesStore()
+  const {data, loading} = useBundles()
```

Comment:
SHOULD: looks like `BundlesOverview.test` is failing since the mock just needs to be updated

Reply:
Thanks for catching! Missed that with all the other failing tests. Will do a spike to fix the tests in the main branch.

Reply:
@jordanl17 PR updated, thanks!